File: csharp/HearthstoneReplays/ReplayData/Meta/Options/Options.cs
#region

using System.Collections.Generic;
using System.Xml.Serialization;

#endregion

namespace HearthstoneReplays.ReplayData.Meta.Options
{
	public class Options : GameData
	{
		[XmlAttribute("id")]
		public int Id { get; set; }

		public List<Option> OptionList { get; set; }
	}
}

File: java/src/main/java/com/hearthsim/hsreplay/parser/replaydata/meta/options/Options.java
package com.hearthsim.hsreplay.parser.replaydata.meta.options;

import java.util.ArrayList;
import java.util.List;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlAttribute;
import javax.xml.bind.annotation.XmlElement;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import com.hearthsim.hsreplay.parser.replaydata.GameData;

@Getter
@Setter
@NoArgsConstructor
@XmlAccessorType(XmlAccessType.FIELD)
@ToString
public class Options extends GameData {

	@XmlAttribute(name = "id")
	private int id;

	@XmlElement(name = "Option")
	private List<Option> optionList = new ArrayList<>();

	public Options(String timestamp, int id, List<Option> optionList) {
		super(timestamp);
		this.id = id;
		this.optionList = optionList;
	}

}
